playbook/app/pb_kits/playbook/pb_selectable_card/_selectable_card.jsx

```diff
@@ -129,6 +129,7 @@ const SelectableCard = ({
                         checked={checked}
                         disabled={disabled}
                         onClick={handleClick}
+                        readOnly
```

Comment:
I'm curious about what this prop might be fixing, but I do understand the HTML attribute. What was happening in this scenario?

Reply:
The kit per @patmccler is using a "visible" input and a "hidden" one, per this `readOnly` on the hidden one.

The warning itself, on the element is essentially asking for either a `readOnly` or an `onChange`. Controlled with the intent of child updates or without.

Reply:
The kit has two inputs. One is hidden and controls the CSS and is present on all variants of the kit
This is the second one, only used for display. When it is clicked, it generates a click on the hidden one which handles `onChange`.
This second input was causing console errors because its given a `checked` attribute and no `onChange` handler. But in reality the `onChange` handler from the hidden input handles any changes

Reply:
Ok cool! TIL 💯  👍